### PR TITLE
update vendor packages for astronomer/astronomer

### PR DESCRIPTION
**Pull request by pgvishnuram**
## Description

image | old | new
--    | --  | --
ap-base | 3.16.2  | 3.16.2-1
ap-db-bootstrapper | 0.26.9 |  0.26.11
ap-nats-server | 2.8.1-2 | 2.8.1-3
ap-nats-exporter | 0.10.0 | 0.10.0-1
ap-pgbouncer-krb | 1.17.0-1 | 1.17.0-3
ap-blackbox-exporter | 0.21.1-2 | 0.22.0 
ap-node-exporter  | 1.3.1 | 1.4.0
ap-prometheus | 2.37.0 | 2.37.1
ap-nats-streaming | 0.24.5-2 | 0.24.5-3
ap-vector | 0.23.0 | 0.23.3-1
ap-kube-state | 2.5.0 | 2.6.0
ap-registry |  3.16.2 | 3.16.2-1
ap-awsesproxy | 1.3-7 | 1.3-8
## Related Issues

related to https://github.com/astronomer/issues/issues/5065
## Testing

QA should see all updated services should function normally without causing any new issues 

## Merging

Do not merge this PR until it lists which release branches this PR should be merged / cherry-picked into.

### CODE REVIEW

**Comment by pgvishnuram** [comment]
@vishwas-astro

**Comment by vishwas-astro** [comment]
@pgvishnuram - get it out of draft mode please

**Comment by vishwas-astro** [comment]
pre-commit is failing?

**Comment by vishwas-astro** [comment]
And mention the GH issue it is pointing to

File: values.yaml [hunk 1]
```diff
@@ -111,7 +111,7 @@ global:
   loggingSidecar:
     enabled: false
     name: sidecar-log-consumer
-    image: quay.io/astronomer/ap-vector:0.23.0
+    image: quay.io/astronomer/ap-vector:0.23.3-1
```
Comment (vishwas-astro) [inline values.yaml:114]: 0.23.0-1 or 0.23.3-1?

**Review by vishwas-astro — APPROVED** [review]

**Pull request merged**